nsxt/resource_nsxt_policy_ip_block.go

```diff
@@ -46,7 +46,7 @@ func resourceNsxtPolicyIPBlock() *schema.Resource {
 			"cidr": {
 				Type:         schema.TypeString,
 				Description:  "Network address and the prefix length which will be associated with a layer-2 broadcast domain",
-				Required:     true,
+				Optional:     true,
```

Comment:
cidr should also be deprecated

Reply:
We mark it as deprecated even though it's the only option for v9.0 and below?